<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="chcnav">
        <!-- hc_cgi_protocol_process_node -->
        <node pkg="chcnav" type="hc_cgi_protocol_process_node" name="hc_topic_driver" output="screen"/>

        <!-- hc_msg_parser_launch_node  -->
        <node pkg="chcnav" type="hc_msg_parser_launch_node" name="c_rs232" output="screen">
            <!-- serial settings -->
            <param name="type" value="serial"/>
            <param name="rate" value="1000"/>         <!-- 节点每秒解析最大协议数量 -->
            <param name="port" value="/dev/ttyUSB0"/> <!-- 串口路径 -->
            <param name="baudrate" value="460800"/>      <!-- 波特率 -->
            <!-- serial settings end -->
        </node>
    </group>
</launch>
